.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "src/Elastic.Markdown"
+    schedule:
+      interval: "weekly"
+    labels:
+      - chore
```

Comment:
Should we use the company renovate instead?

https://github.com/elastic/docs-builder/pull/1

Reply:
Discussed offline, we are going with dependabot!